airbyte-oauth/src/main/java/io/airbyte/oauth/flows/SalesforceOAuthFlow.java

```diff
@@ -20,32 +20,32 @@
 
 /**
  * Following docs from
- * https://help.salesforce.com/s/articleView?language=en_US&id=sf.remoteaccess_oauth_web_server_flow.htm
+ * https://help.salesforce.com/s/articleView?language=en_US&amp;id=sf.remoteaccess_oauth_web_server_flow.htm
```

Comment:
Is this right? Should it replace the ampersand itself?

It's also a bit sad that this link won't work from IntelliJ anymore. Would it be possible to just make this comment not a javadoc by removing the second * at the start?

Reply:
Yeah we can make this a regular comment.  Javadoc is required to conform to html escaping - this was causing a compile error on the javadoc.  The only option is to make it a regular comment instead (or as well).